devops/azure-pipelines/azure-pipelines.yml

```diff
@@ -0,0 +1,14 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+trigger:
+- main
```

Comment:
we also presumably want builds on dev branch

Reply:
also, add an explicit `pr:` configuration, even if you want to keep the default behavior, because it's the kind of thing that should be explicit.

Reply:
Changes made.